sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -136,10 +137,6 @@ case class GpuTimeSub(
 
   override lazy val resolved: Boolean = childrenResolved && checkInputDataTypes().isSuccess
 
-  override def withTimeZone(timeZoneId: String): TimeZoneAwareExpression = {
```

Comment:
why did this move?

Reply:
I had to move it because the GpuTimeMath is an abstract class. copy function is only available for case classes. I wish I could extend from a case class but that's not advised. Any other way you can think of doing this?

Reply:
no thats fine.